reporter/gen_report.js

```diff
@@ -32,11 +35,22 @@ Object.keys(originals).forEach(key => {
       escape(atob(originals[key]))));
 });
 
+var fileWriter = new FileWriter(false);
+
+fileWriter.writeFile = function writeFile(file, callback) {
+  if (file.length > 255) {
+    var excess = file.length - 255;
+    var baseName = path.basename(file, '.html')
```

Comment:
But those normally don't hit the 255 chars length, but I can also use the `path.extname()`.

Will do.